lib/components/FormList.js

```diff
@@ -52,18 +54,29 @@ function MLFormList$init () {
     this.once('childrenbound', onChildrenBound);
 }
 
+function MLFormList$setItemSchema (schema) {
+    this._subFormSchema = schema.subSchema;
+    this._movable = !!schema.allowMove;
+    this._deletable = !!schema.allowDelete;
+    this._itemLabel = schema.itemLabel;
+}
+
 function MLFormList$moveItem (fromIndex, toIndex) {
     const toInsert = this.model.m.splice(fromIndex, 1);
     if (toInsert) return this.model.m.splice(toIndex, 0, toInsert[0]);
 }
 
+function MLFormList$destroy () {
+    if (this._connector) milo.minder.destroyConnector(this._connector);
```

Comment:
The connector var (initialised on line 77) has two underscore characters, not one.